chart/templates/_helpers.tpl

```diff
@@ -187,6 +187,15 @@ The duckdb-index/ subpath in the NFS
 {{- printf "%s/%s/%s/" .Chart.Name .Release.Name "duckdb-index" }}
 {{- end }}
 
+{{/*
+The stats-cache/ subpath in the NFS
```

Comment:
@severo  well we need to have  this subdirectory to EFS as well as in https://github.com/huggingface/datasets-server/pull/1495?

Reply:
could it be in the same volume as `duckdb-index` (`duckdb-data` if I understand it right...)?

Reply:
I've created (hopefully correctly... ?) a separate volume for now, though don't know if it's worth it cc @severo

Reply:
OK. I think it's a good idea to have created a separate volume. Let's monitor, and if it proves to be overkill, we can merge afterward.